securityscanutils/trivy_scanner_test.go

```diff
@@ -56,7 +56,7 @@ var _ = Describe("Trivy Scanner", func() {
 		})
 		completed, vulnFound, err := t.ScanImage(context.TODO(), inputImage, inputMarkdownTemplateFile, outputFile)
 
-		Expect(err).NotTo(HaveOccurred())
+		Expect(err).To(MatchError(ImageNotFoundError))
```

Comment:
Do we have any unit tests that check more than 1 image at a time to ensure it's continuing after the error?

Reply:
I added a missing image name to an existing test suite that asserts the generated files for all images exist after the scan completes.

Reply:
Nice. If we can confirm the generated files contain the later images "gloo" and "discovery", that would be nice, but I'm not familiar enough with the test suite to know how much of a pain that would be.

Reply:
For reference, this is covered by this assertion in `securityscan_test.go`

```go
ExpectDirToHaveFiles(path.Join(markdownDir, "1.7.0"), "discovery_cve_report.docgen", "gloo_cve_report.docgen")
```

The `.docgen` files will only exist if they were scanned.

Reply:
The `.docgen` files will only exist is _something_ was scanned, right? But having "gloo" and "discovery" specifically confirms that it continues after "thisimagecannotbefound", unless I misunderstood.

Though seeing the recent update to the v2.4.6 Security Alert issue, I'm wondering if order isn't maintained anyway.